xgcm/test/test_grid.py

```diff
@@ -784,20 +784,62 @@ def test_boundary_kwarg_same_as_grid_constructor_kwarg():
 
 @pytest.mark.parametrize(
     "metric_axes,metric_name",
-    [("X", "dx_t"), ("Y", "dy_ne"), (["Y", "X"], "dy_n")],
+    [(["Y", "X"], "area_n")],
 )
-def test_interp_like(metric_axes, metric_name):
+@pytest.mark.parametrize("periodic", [True, False])
+@pytest.mark.parametrize(
+    "boundary, boundary_expected",
+    [
+        ({"X": "fill", "Y": "fill"}, {"X": "fill", "Y": "fill"}),
+        ({"X": "extend", "Y": "extend"}, {"X": "extend", "Y": "extend"}),
+        (
+            {"X": "extrapolate", "Y": "extrapolate"},
+            {"X": "extrapolate", "Y": "extrapolate"},
+        ),
+        ("fill", {"X": "fill", "Y": "fill"}),
+        ("extend", {"X": "extend", "Y": "extend"}),
+        ("extrapolate", {"X": "extrapolate", "Y": "extrapolate"}),
+        ({"X": "extend", "Y": "fill"}, {"X": "extend", "Y": "fill"}),
+        ({"X": "extrapolate", "Y": "fill"}, {"X": "extrapolate", "Y": "fill"}),
+        pytest.param(
```

Comment:
This set of parameters causes the test to fail under `periodic=True` (therefore pytest.mark.xfail works) but passes under `periodic=False`

Reply:
Maybe create a separate test for those expected to fail